### PR TITLE
(blog): Integrate empress-blog and Netlify CMS

### PR DESCRIPTION
**Pull request by abhilashlr**

### CODE REVIEW

File: source/2020-06-05-the-ember-times-issue-151.md [hunk 1]
```diff
@@ -104,14 +105,15 @@ Read more on the full the blog posts on [using Oauth2 with Flask-REST-JSONAPI](h
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Blog post: How to integrate Netlify-CMS with Empress-blog 🚀](https://www.mylittletechlife.com/integrate-netlify-cms-with-empress-blog)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Robin Philip Thomas](https://github.com/MalayaliRobz) wrote a blog on integrating empress-blog and Nelify CMS.
```
Comment (ijlee2) [inline source/2020-06-05-the-ember-times-issue-151.md]:
```suggestion
[Robin Philip Thomas](https://github.com/MalayaliRobz) wrote a blog on integrating empress-blog and Netlify CMS.
```

File: source/2020-06-05-the-ember-times-issue-151.md [hunk 2]
```diff
@@ -104,14 +105,15 @@ Read more on the full the blog posts on [using Oauth2 with Flask-REST-JSONAPI](h
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+[empress-blog](https://github.com/empress/empress-blog) is designed to be a fully-functional, static site implementation of a blog system built on Ember with fully working out of the box SEO friendly output based on markdown. [Netlify CMS](https://www.netlifycms.org/) is based on client-side javascript, reusable APIs and prebuilt markup.
```
Comment (ijlee2) [inline source/2020-06-05-the-ember-times-issue-151.md]: I'm not too familiar with Empress so I wasn't sure how to parse the first sentence. I wondered if making a list might help clarify what its features are.

If I understand you right, maybe:
```suggestion
What is [empress-blog](https://github.com/empress/empress-blog)?

- A fully-functional, static site implementation of a blog system
- Built on Ember
- Fully working out of the box
- SEO friendly output based on Markdown

[Netlify CMS](https://www.netlifycms.org/) is based on client-side JavaScript, reusable APIs and prebuilt markup.
```

**Review by ijlee2 — APPROVED** [review]
Thanks for covering Robin's blog post! I made a couple of suggestions. Will you take a look?

File: source/2020-06-05-the-ember-times-issue-151.md (hunk 2 above)
Comment (abhilashlr) [reply source/2020-06-05-the-ember-times-issue-151.md]: Ah! That sounds perfect @ijlee2 :)

**Comment by abhilashlr** [comment]
> Thanks for covering Robin's blog post! I made a couple of suggestions. Will you take a look?

Thanks for the review comments @ijlee2 :) I've made the changes you have suggested 👍

**Comment by ijlee2** [comment]
@abhilashlr Thanks! After applying a reviewer's feedback (feel free to decline suggestions too, because I do want you to write what you think sounds good), you are welcome to merge your own PR. I don't think we're strict on who merges a PR. ☺️

I can go ahead and merge this for you.

**Comment by abhilashlr** [comment]
> @abhilashlr Thanks! After applying a reviewer's feedback (feel free to decline suggestions too, because I do want you to write what you think sounds good),

Sure @ijlee2 but that suggestion was explaining in pointers much better than single line of statements. 

>  you are welcome to merge your own PR. I don't think we're strict on who merges a PR. ☺️ 
> I can go ahead and merge this for you.

Sure I will do that from next time once it is reviewed :)

**Pull request merged**